AsyncDisplayKit/ASCellNode.mm

```diff
@@ -117,15 +118,18 @@ - (void)layoutDidFinish
   _viewControllerNode.frame = self.bounds;
 }
 
-- (void)__setNeedsLayout
+- (void)displayNodeDidInvalidateSizeOldSize:(CGSize)oldSize
 {
-  CGSize oldSize = self.calculatedSize;
-  [super __setNeedsLayout];
-  
-  //Adding this lock because lock used to be held when this method was called. Not sure if it's necessary for
-  //didRelayoutFromOldSize:toNewSize:
-  ASDN::MutexLocker l(__instanceLock__);
-  [self didRelayoutFromOldSize:oldSize toNewSize:self.calculatedSize];
+  ASSizeRange constrainedSize = ASSizeRangeMake(CGSizeZero, CGSizeMake(CGRectGetWidth(self.bounds), CGFLOAT_MAX));
```

Comment:
Let's remove the default constrained size for now. From an optimization standpoint we should pass in the constrained size or the new layout in this method in the future so we don't have to recalculate.